src/justenoughicu.c

```diff
@@ -28,24 +29,32 @@ int icu_case(lua_State *L) {
   u_strFromUTF8(input_as_uchar, l, &l, input, input_l, &err);
 
   /* Now do the conversion */
-  conversion_function_t conversion;
   UChar *output;
   int32_t l2 = 0;
 
-  if (strcmp(recase, "upper") == 0) {
-    conversion = u_strToUpper;
-  } else if (strcmp(recase, "lower") == 0) {
-    conversion = u_strToLower;
-  // } else if (strcmp(recase, "title") == 0) {
-    // conversion = u_strToTitle;
+  if (strcmp(recase, "title") == 0) {
+    conversion_function_ti conversion;
+    UBreakIterator *iter;
```

Comment:
at least you want `iter=NULL;` here. That's why it blew up. see [docs](http://icu-project.org/apiref/icu4c/ustring_8h.html#a8c6202ad4ac954e225aae061b184e450)

Side note: This is the oddest way to call ICU I've seen yet… you're including `ubrk.h`, why not just call `u_strToTitle` and dispense with the typedef?